presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/SymbolMapper.java

```diff
@@ -64,6 +64,13 @@ public Symbol map(Symbol symbol)
         return canonical;
     }
 
+    public List<Expression> map(List<Expression> values)
```

Comment:
I don't see a change to SymbolMapper in the "Support ValuesNode in SymbolMapper" commit.  Is it missing code, or am I misunderstanding something?